bootstrap/js/_ecosystem.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+$(document).ready(function () {
```

Comment:
Vanilla JavaScript is good in 2024 and it should be dropped when we move to Bootstrap 5.

Reply:
Fair enough, I saw jquery was already added as a dependency and went with it, but I'll go ahead and refactor